core/integration/features/verified/0070-MKTD-market-decimal-places-negative.feature

```diff
@@ -0,0 +1,176 @@
+            | market.liquidity.bondPenaltyParameter         | 0.2   |
+            | market.liquidity.targetstake.triggering.ratio | 0.1   |
+        And the following assets are registered:
+            | id  | decimal places |
+            | ETH | 5              |
+            | USD | 2              |
+        And the average block duration is "1"
+        And the log normal risk model named "log-normal-risk-model-1":
+            | risk aversion | tau | mu | r | sigma |
+            | 0.000001      | 0.1 | 0  | 0 | 1.0   |
+        #risk factor short: 3.5569036
+        #risk factor long: 0.801225765
+
+        And the fees configuration named "fees-config-1":
+            | maker fee | infrastructure fee |
+            | 0.004     | 0.001              |
+        And the price monitoring named "price-monitoring-1":
+            | horizon | probability | auction extension |
+            | 3600    | 0.99        | 300               |
+        And the markets:
+            | id        | quote name | asset | risk model              | margin calculator         | auction duration | fees          | price monitoring   | oracle config          | decimal places | position decimal places |
+            | ETH/MAR22 | ETH        | USD   | log-normal-risk-model-1 | default-margin-calculator | 1                | fees-config-1 | price-monitoring-1 | default-eth-for-future | 0              | 0                       |
+            | USD/DEC19 | USD        | ETH   | log-normal-risk-model-1 | default-margin-calculator | 1                | default-none  | price-monitoring-1 | default-usd-for-future | 3              | 3                       |
+            | USD/DEC20 | USD        | ETH   | log-normal-risk-model-1 | default-margin-calculator | 1                | default-none  | price-monitoring-1 | default-usd-for-future | 5              | 5                       |
+            | USD/DEC21 | USD        | ETH   | log-normal-risk-model-1 | default-margin-calculator | 1                | default-none  | price-monitoring-1 | default-usd-for-future | 5              | 3                       |
+            | USD/DEC22 | USD        | ETH   | log-normal-risk-model-1 | default-margin-calculator | 1                | default-none  | price-monitoring-1 | default-usd-for-future | 5              | -1                      |
+        And the parties deposit on asset's general account the following amount:
+            | party  | asset | amount    |
+            | party0 | USD   | 5000000   |
+            | party0 | ETH   | 5000000   |
+            | party1 | USD   | 100000000 |
+            | party1 | ETH   | 100000000 |
+            | party2 | USD   | 100000000 |
+            | party2 | ETH   | 100000000 |
+            | party3 | USD   | 100000000 |
+            | lpprov | ETH   | 100000000 |
+            | lpprov | USD   | 100000000 |
+
+    Scenario: 001, Markets with different precisions trade at the same price
```

Comment:
Would it be worth checking the mark price; since that's somehow what the description implies?